src/test/java/org/mockito/internal/matchers/text/ValuePrinterTest.java

```diff
@@ -14,7 +16,22 @@ public void prints_values() throws Exception {
         assertEquals("null", print(null));
         assertEquals("\"str\"", print("str"));
         assertEquals("\"x\ny\"", print("x\ny"));
+        assertEquals("3", print(3));
+        assertEquals("3L", print(3L));
+        assertEquals("3.14d", print(3.14d));
+        assertEquals("3.14f", print(3.14f));
         assertEquals("[1, 2]", print(new int[]{1, 2}));
+        assertEquals("{\"hoge\" = 2L}", print(new LinkedHashMap<String, Object>() {
```

Comment:
@tokuhirom 
for better readability better use [Collections.singletonMap(K,V)](https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#singletonMap(K,%20V)) here